friendsofphp/php-cs-fixer/2.2/.php_cs.dist

```diff
@@ -1,8 +1,14 @@
 <?php
 
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
```

Comment:
Why not 
```php
->in([
    __DIR__.'/src/',
    __DIR__.'/tests/',
])
```
Much more explicit, and by default excludes var and vendors

Reply:
I like your proposal @Pierstoval 👌🏻

Reply:
I agree too